src/server_manager/electron_app/start.action.sh

```diff
@@ -16,9 +16,8 @@
 
 run_action server_manager/electron_app/build
 
-readonly NODE_MODULES_BIN_DIR="${ROOT_DIR}/src/server_manager/node_modules/.bin"
-
 cd "${BUILD_DIR}/server_manager/electron_app/static"
+
 OUTLINE_DEBUG='true' \
-SB_METRICS_URL='https://dev.metrics.getoutline.org' \
-"${NODE_MODULES_BIN_DIR}/electron" .
+    SB_METRICS_URL='https://dev.metrics.getoutline.org' \
```

Comment:
The shell style guide uses 2 spaces for identation: https://google.github.io/styleguide/shellguide.html#s5.3-pipelines
https://google.github.io/styleguide/shellguide.html#indentation

Reply:
oh, good